app/src/main/java/com/android/sample/model/ingredient/DefaultIngredientRepository.kt

```diff
@@ -0,0 +1,89 @@
+package com.android.sample.model.ingredient
+
+import com.android.sample.model.ingredient.localData.IngredientLocalRepository
+import com.android.sample.model.ingredient.networkData.IngredientNetworkRepository
+
+/**
+ * Default implementation of the IngredientRepository interface.
+ *
+ * @property localRepository The local repository for ingredient data.
+ * @property networkRepository The network repository for ingredient data.
+ */
+open class DefaultIngredientRepository(
```

Comment:
Yes!

Reply:
Ok, I think in the future we will have to solve this problem but since it is not that big of an issue we can for the moment leave it as it is 👍, thanks for the explanation